Fixing Issue #829

Bernasss12 opened the pull request:
### Trying to fix issue #829
`GuiRequestTable.java` has no hardcoded text.

Please review this commit and confirm/deny if this was the basis of the issue or whether this is to be done or not. 

Also please review the `en_us.lang` file formatting and correct me in any way you see fit.

Thanks for the attention.

Commit:
Fixing Issue #829

Trying to fix issue #829
GuiRequestTable.java has no hardcoded text.

Please review this commit and confirm/deny if this was the basis of the issue or whether this is to be done or not. Thanks for the attention.

bziemons commented on resources/assets/logisticspipes/lang/en_us.lang:
```diff
@@ -323,6 +323,54 @@ lp.hud.config.satellite.enabled=Enabled Satellite.
+
+
+
+
```
empty lines?

Bernasss12 replied:
Bad habits, I don't like to see text below where I'm writing but I'll make sure to always remove it before committing from now on.

bziemons commented:
I don't know if we want to translate symbols.. It isn't bad, but it would make language files quite a bit bigger. Other than that you are on the right track.

Bernasss12 commented:
I was actually skeptical if those were indeed necessary to change since, even without thinking about language file size, I can't thing of a single reason to change it in other languages, and if changed it would probably only actually break the GUI. I'll fix that and not change these cases from now on.

About the lang file comments/organization/naming anything to point out?

bziemons commented:
That's fine :)

> About the lang file comments/organization/naming anything to point out?

Looks good. Just keep in mind to re-use the translations if applicable and rename the keys to something more general (in that case) :)

Bernasss12 commented:
I had that in mind, in fact the class I finished can pretty much have everything re-used for the Request Pipes :)
But thanks for pointing out!

Commit:
Progress

Fully localized:
- `gui/orderer/FluidGuiOrderer.java`
- `gui/orderer/GuiOrderer.java`
- `gui/orderer/GuiRequestTable.java`
- `gui/orderer/NormalGuiOrderer.java`
- `gui/orderer/NormalMK2GuiOrderer.java`
- `utils/gui/ItemDisplay.java`

Added TODO in:
- `utils/string/StringUtils.java`

davboecki commented on resources/assets/logisticspipes/lang/en_us.lang:
```diff
@@ -322,16 +322,48 @@ lp.hud.config.provider.disabled=Disabled Provider.
+gui.request.textMissing=Missing
+gui.request.textComponents=Components
+# - ItemDisplay
+gui.item_displayp.age=Page
```
typo?

davboecki commented on common/logisticspipes/utils/gui/ItemDisplay.java:
```diff
@@ -271,9 +284,12 @@ public void renderItemArea(double zLevel) {
-			buf.pos(xPosition + 100, yPosition + 100, zLevel).tex(0.08, 0.72 + (graphic * 0.03125)).endVertex();;
-			buf.pos(xPosition + 100, yPosition, zLevel).tex(0.08, 0.69 + (graphic * 0.03125)).endVertex();;
+			buf.pos(xPosition, yPosition + 100, zLevel).tex(0.04, 0.72 + (graphic * 0.03125)).endVertex();
+			;
```
Instead of creating new lines here, just remove them, please.

Bernasss12 replied:
I don't really know what happened here but I'm guessing it was the auto formatting

Bernasss12 replied:
But yes Ill remove the semicolons

Bernasss12 commented:
I already changed that later on but yes that seems to be a typo

On Thu, 13 Jun 2019, 22:22 davboecki, <notifications@github.com> wrote:

> *@davboecki* commented on this pull request.
> ------------------------------
>
> In resources/assets/logisticspipes/lang/en_us.lang
> <https://github.com/RS485/LogisticsPipes/pull/1248#discussion_r293582422>:
>
> > +gui.request.buttonShow=Show
> +gui.request.buttonDisk=Disk
> +gui.request.buttonMore=more
> +# -- Text
> +gui.request.textPopup=Popup
> +gui.request.textList=List
> +gui.request.textUpperID=ID
> +gui.request.textRequestType=Request Type
> +gui.request.textSendRouterID=Send to Router ID
> +gui.request.textRequestID=Request ID
> +gui.request.textYouAreMissing=You are missing:
> +gui.request.textRequest_success=Request successful!
> +gui.request.textMissing=Missing
> +gui.request.textComponents=Components
> +# - ItemDisplay
> +gui.item_displayp.age=Page
>
> typo?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/RS485/LogisticsPipes/pull/1248?email_source=notifications&email_token=ABPR3HGQ5MVWLWL2Z3TNLBDP2K3CHA5CNFSM4HXEKBJKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOB3QMRCY#pullrequestreview-249612427>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABPR3HHL7FNQE5D626JELMLP2K3CHANCNFSM4HXEKBJA>
> .
>

davboecki commented:
Other than the two comments I added everything else lgtm.

Bernasss12 commented:
I will restart my branch for this issue since there has been a lot of updates since I started this.

bziemons commented:
just rebase and you'll be fine

Bernasss12 commented:
It's much easier and less prone to error this way for me
